opds_import.py

```diff
@@ -275,7 +276,15 @@ class OPDSImporter(object):
 
     COULD_NOT_CREATE_LICENSE_POOL = (
         "No existing license pool for this identifier and no way of creating one.")
-   
+
+    NAME = ExternalIntegration.OPDS_IMPORT
+    FIELDS = [
+        {
+            "key": Collection.EXTERNAL_ACCOUNT_ID_KEY,
+            "label": _("URL"),
+        }
+    ]
+
```

Comment:
This isn't a problem with this branch, but to understand this change I went looking in opds_import.py and I think it's a little confusing that `SimplifiedOPDSLookup` looks for the URL of the OPDS feed in `integration.url` and `OPDSImporter` looks in `integration.external_account_id`. Do you think this is worth changing?